### DISCUSSION TITLE
Slow prompt processing

### DISCUSSION THREAD

**Discussion by CeciliaXCIX** [YellowRoseCx/koboldcpp-rocm#72] [General]
The prompt processing is incredibly slow until I restart my computer for some reason. I'm not sure what causes this and I'm just curious if anyone else is having this issue. Happens on all model sizes. I'm using a 7900xtx.
If anyone has a potential fix, that'd also be cool.

**Comment by YW5555** [comment]
Check to see if your VRAM speed is stuck at 0mhz, it's a power saving feature on my 6900XT and KoboldCpp doesn't 'wake' the VRAM during generation.

**Reply by CeciliaXCIX** [reply]
It doesn't get stuck at 0, but it is quite low; around 900MHz.

**Reply by CeciliaXCIX** [reply]
Did you ever find a way to disable the power saving feature?

**Reply by YW5555** [reply]
Turning off free sync seems to disable it by keeping the VRAM at full speed. But your idle power draw will be higher.

**Reply by CeciliaXCIX** [reply]
Hmm, I'll give it a shot

**Reply by CeciliaXCIX** [reply]
Doesn't seem to change anything for me unfortunately.

**Comment by CeciliaXCIX** [comment]
Nvm the issue is still there. I'm unsure how to fix this.
